### PR TITLE
Completed the homepage

### PR DESCRIPTION
**Pull request by aryan-13**
-Structured the code
-navbar, header, footer completed
-started with the documentation page

Here is a preview: 
landing-page

### CODE REVIEW

**Comment by anshikaa12** [comment]
folder structure looks organised and overall code looks good

File: css/base/typography.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.heading--1 {
```
Comment (ritik-patel05) [inline css/base/typography.css:1]: Maybe shorten the class name to h-1

File: css/base/typography.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	font-size: 4rem;
+	font-weight: 800;
+	display: block;
+	margin: 1rem;
+	letter-spacing: 0.1rem;
+}
+.heading--2 {
```
Comment (ritik-patel05) [inline css/base/typography.css:8]: can be h-2?

File: css/base/utilities.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.u-center-text {
+	text-align: center !important;
+}
+
+.u-margin-bottom-small {
```
Comment (ritik-patel05) [inline css/base/utilities.css:5]: can be shortened to u-mb-sm

File: css/base/utilities.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	margin-bottom: 1.2rem !important;
+}
+.u-margin-bottom-medium {
```
Comment (ritik-patel05) [inline css/base/utilities.css:8]: u-mb-md

File: css/base/utilities.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+	margin-bottom: 4rem !important;
+}
+.u-margin-bottom-big {
```
Comment (ritik-patel05) [inline css/base/utilities.css:11]: u-mb-big

File: css/base/utilities.css [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+	margin-bottom: 8rem !important;
+}
+
+.u-margin-top-big {
```
Comment (ritik-patel05) [inline css/base/utilities.css:15]: u-mt-big

File: css/base/utilities.css [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+	margin-top: 8rem !important;
+}
+.u-margin-top-huge {
```
Comment (ritik-patel05) [inline css/base/utilities.css:18]: u-mt-huge/lg (denoting large)

**Review by ritik-patel05 — COMMENTED** [review]
Maybe use shorter class names? What do you think?
1. margin => m
2. padding => p
3. margin-bottom => mb, etc.
4. small => sm, large => lg,

File: css/pages/documentation.css [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+}
+.content3 {
+	background-color: orange;
+	grid-area: content3;
+}
+.doc-footer {
+	background-color: yellow;
+	grid-area: footer;
+}
+@media only screen and (max-width: 550px) {
```
Comment (Bharatrathod-81) [inline css/pages/documentation.css:41]: awesome work

Comment (Bharatrathod-81) [inline css/pages/documentation.css:41]: awesome work

**Pull request merged**